lib/tools/xctest-commands.js

```diff
@@ -0,0 +1,51 @@
+import os from 'os';
```

Comment:
can we have functional tests for these commands?

Reply:
I will add some tomorrow. I need to add the WDA bundle into the repo as assets